src/python/bot/fuzzers/syzkaller/generate_config.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate Syzkaller fuzzer config dynamically."""
+
+import json
+import os
+
+
+def run(serial,
```

Comment:
Would prefer for this module to be named "config", and this method "generate".

i.e. config.generate()

Reply:
Done.